ops/terraform/services/static-site/README.md

```diff
@@ -0,0 +1,91 @@
+# `static-site` Terraservice
+
+This Terraservice contains the Terraform IaC for BFD's static-site documentation infrastructure. 
+
+## Usage
+
+The `base` Terraservice _must_ be applied, to the _same Terraform workspace_, prior to `apply`ing this Terraservice. This Terraservice relies on the resources created by that Terraservice, and will fail to apply if they do not exist.
+
+Assuming you have created a Terraform workspace corresponding to your target environment and have switched to it, this Terraservice can be applied without specifying any variables using:
+
+```bash
+terraform apply
+```
+
+<!-- BEGIN_TF_DOCS -->
+<!-- GENERATED WITH `terraform-docs .`
+     Manually updating the README.md will be overwritten.
+     For more details, see the file '.terraform-docs.yml' or
+     https://terraform-docs.io/user-guide/configuration/
+-->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | ~> 5.53.0 |
+
```

Comment:
Looks like the `terraform-docs` header is missing. Mind re-adding it and re-running `terraform-docs .` to update the `README`?

```suggestion

<!-- BEGIN_TF_DOCS -->
```

Reply:
I have no idea to what header you refer, please elaborate.

Reply:
Sorry, looks like I may have pulled an old version of the `README.md` or this has since been corrected. The header's there and runnning `terraform-docs .` yields no changes. I'll resolve this conversation.